app/assets/v2/js/grants/funding.js

```diff
@@ -31,13 +31,6 @@ $(document).ready(function() {
       $('#tweetModal a.button').attr('href', 'https://twitter.com/intent/tweet?text=I%20just%20funded%20these%20' + donations.length + '%20grants%20on%20@gitcoin%20=%3E%20' + bulk_add_cart);
     }
     CartData.setCart([]);
-
-    $(document).keydown(function(e) {
```

Comment:
why remove this?  seems unrelated to wallet stuff

Reply:
because I fixed the modal to use properly with bootstrap modal, in that way modal scroll get fixed, pressing ESC close the modal without need to create that functionality also you will find $('#tweetModal').modal('show'); and a new class to make the modal scrollable , that is all from bootstrap modals.

Reply:
Im fixing the wallet but with that finding a few things broken, like send.js and tokens.js injected more that 2 times in a few pages like townsquare and grants details.

Reply:
updated the description. 
Doing some regression test before I put it ready for review.